docs/Changelog.md

```diff
@@ -8481,6 +8481,56 @@ This version of the operator has been available since version 9 of the default O
 <dd>Constrain input and output types to float tensors.</dd>
 </dl>
 
+### <a name="Cast-9"></a>**Cast-9**</a>
+
+  The operator casts the elements of a given input tensor to a data type
+  specified by the 'to' argument and returns an output tensor of the same size in
+  the converted type. The 'to' argument must be one of the data types specified
+  in the 'DataType' enum field in the TensorProto message.
+  
+  Casting from string tensor in plain (e.g., "3.14" and "1000") and scientific numeric representation
```

Comment:
```suggestion
  Casting from string tensor in plain (e.g., "3.14" and "1000") and scientific numeric representations
```

Reply:
ooops.. clicked the auto correction.. should do fix in .cc source